### PR TITLE
Category page layout

### PR DESCRIPTION
**Pull request by eldiegod**
fixes:  #1359

### CODE REVIEW

File: src/views/viewer/CategoryView/CategoryView.tsx [hunk 1]
```diff
@@ -27,9 +39,24 @@ const dummyHeroVideos = [
 ]
 
 export const CategoryView = () => {
-  const { videos } = useVideos({ limit: 3 })
+  const MdBreakpointMatch = useMediaMatch('md')
```
Comment (kdembler) [inline src/views/viewer/CategoryView/CategoryView.tsx]: shouldn't be upper-cased

**Comment by kdembler** [comment]
we also need some empty state when filters produce no result (also the header is broken in that case):
Screenshot 2021-10-21 at 16 51 42

File: src/views/viewer/CategoryView/CategoryView.style.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  padding-bottom: ${sizes(6)};
+  border-bottom: 1px solid ${colors.gray[700]};
+`
+
+export const CategoriesContainer = styled(LayoutGrid)`
+  margin-top: ${sizes(12)};
+`
+
+export const FallbackWrapper = styled.div`
+  /* margin: ${sizes(46)} auto ${sizes(31)} auto; */
```
Comment (kdembler) [inline src/views/viewer/CategoryView/CategoryView.style.ts]: What's the purpose of this?

File: src/views/viewer/CategoryView/CategoryView.style.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+  ${media.sm} {
+    margin: ${sizes(46)} auto ${sizes(31)} auto;
```
Comment (kdembler) [inline src/views/viewer/CategoryView/CategoryView.style.ts]: These shouldn't be expressed with `sizes`

File: src/views/viewer/CategoryView/CategoryVideos.tsx [hunk 4]
```diff
@@ -64,8 +62,8 @@ export const CategoryVideos: React.FC<{ categoryId: string }> = ({ categoryId })
   return (
     <Container>
       <ControlsContainer>
-        <GridItem colSpan={{ base: 2, md: 1 }}>
-          <Text variant={mdMatch ? 'h4' : 'h5'}>All videos ({videoCount})</Text>
+        <GridItem colSpan={{ base: 2, sm: 1 }}>
+          <Text variant={mdMatch ? 'h4' : 'h5'}>All videos {videoCount !== undefined && `(${videoCount})`}</Text>
         </GridItem>
         <StyledSelect
           placeholder="Select language"
```
Comment (kdembler) [inline src/views/viewer/CategoryView/CategoryVideos.tsx]: This should be `Any language`

File: src/views/viewer/CategoryView/CategoryView.style.ts (hunk 2 above)
Comment (eldiegod) [reply src/views/viewer/CategoryView/CategoryView.style.ts]: is to adjust the margins around the Fallback component for desktop as styling the component directly didn't work out too well

Comment (kdembler) [reply src/views/viewer/CategoryView/CategoryView.style.ts]: I was asking about the comment but I see it's been removed since

**Review by kdembler — APPROVED** [review]

**Pull request merged**